cmd/state-svc/test/integration/svc_int_test.go

```diff
@@ -0,0 +1,148 @@
+package integration
+
+import (
+	"net"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"syscall"
+	"testing"
+	"time"
+
+	"github.com/ActiveState/cli/internal/constants"
+	"github.com/ActiveState/cli/internal/exeutils"
+	"github.com/ActiveState/cli/internal/fileutils"
+	"github.com/ActiveState/cli/internal/svcctl"
+	"github.com/ActiveState/cli/internal/testhelpers/e2e"
+	"github.com/ActiveState/cli/internal/testhelpers/tagsuite"
+	"github.com/shirou/gopsutil/process"
+	"github.com/stretchr/testify/suite"
+)
+
+type SvcIntegrationTestSuite struct {
+	tagsuite.Suite
+}
+
+func (suite *SvcIntegrationTestSuite) TestStartStop() {
+	suite.OnlyRunForTags(tagsuite.Service)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.SpawnCmdWithOpts(ts.SvcExe, e2e.WithArgs("status"))
+	cp.Expect("Service cannot be reached")
+	cp.ExpectExitCode(1)
+
+	cp = ts.SpawnCmdWithOpts(ts.SvcExe, e2e.WithArgs("start"))
+	cp.Expect("Starting")
+	cp.ExpectExitCode(0)
+	time.Sleep(500 * time.Millisecond) // wait for service to start up
+
+	cp = ts.SpawnCmdWithOpts(ts.SvcExe, e2e.WithArgs("status"))
+	cp.Expect("Checking")
+
+	// Verify it created a socket file.
+	sockFile := svcctl.NewIPCSockPathFromGlobals().String()
+	suite.True(fileutils.TargetExists(sockFile))
+
+	// Verify the server is running on its reported port.
+	cp.ExpectRe("Port:\\s+:\\d+\\s")
+	portRe := regexp.MustCompile("Port:\\s+:(\\d+)")
+	port := portRe.FindStringSubmatch(cp.TrimmedSnapshot())[1]
+	_, err := net.Listen("tcp", "localhost:"+port)
+	suite.Error(err)
+
+	// Verify it created and wrote to its reported log file.
+	cp.ExpectRe("Log:\\s+.+?\\.log")
+	logRe := regexp.MustCompile("Log:\\s+(.+?\\.log)")
+	logFile := logRe.FindStringSubmatch(cp.TrimmedSnapshot())[1]
+	suite.True(fileutils.FileExists(logFile))
```

Comment:
These kind of assertions show up as "Should be true" when they fail. Which gives zero insight into what actually failed. You then have to look at the stack and actual code to find out what went wrong.

Please at minimum at a description that explains what it was asserting here. Though I believe there is an actual `FileExists` assertion.

This goes for any true/false assertions.